big_tests/tests/mod_http_notification_SUITE.erl

```diff
@@ -14,6 +14,8 @@
 -include_lib("common_test/include/ct.hrl").
 -include_lib("eunit/include/eunit.hrl").
 
+-define(ETS_TABLE, mod_http_notification).
```

Comment:
What is the motivation for using this ETS table instead of sending the message with the result?

Reply:
It follows the example from `service_mongoose_system_metrics_SUITE`. The problem of passing the pid is that the listener cannot then be started until the `init_per_testcase`, where we actually know the pid of the process that will run the test. And in that case, the pool can't either. So tests still can't be parallelised (unless we start pools for different hosts and tests register their clients with different hosts as well?), and it makes the whole listener-pool initialisation and termination run too often.

But actually, this was the intention in the other test suites. In the case of this one, it all still runs on the init_per_testcase, that is true 🤔 So perhaps here it is like this just for consistency? I could revert this specific change to how it was before, using pids and messages 🤷‍♂️

Reply:
Thanks for the explanation, I don't think it needs to be reverted.